bpf-gpl/tc.c

```diff
@@ -455,6 +455,13 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 		}
 	}
 
+	// Drop packets from/to workload Endpoints with IP options
+	if (CALI_F_TO_WEP || CALI_F_FROM_WEP) {
+		if (ip_header->ihl > 5) {
+			CALI_DEBUG("Drop packets with IP options\n");
+			return TC_ACT_SHOT;
```

Comment:
```suggestion
			goto deny;
```

Just to send it through the common path. We should also set `fwd.reason = ` the reasons are in `reasons.h` and we should add a new reason if none applies (most likely)